text/0053-enum-entities.md

```diff
@@ -0,0 +1,146 @@
+# Enumerated Entity Types
+
+## Timeline
+
+- Started: 2024-02-20
+
+## Summary
+
+Extend schemas to support declared enumerations of entity-typed values, analogous to how schemas can currently be used to enumerate a finite list of `Action`-typed values.
+
+## Basic example
+
+An enumerated entity type is declared as a normal entity type, but includes the keyword `enum` followed by the list of legal entity UIDs. Here is a simple example:
+```
+entity User;
+entity Color enum ["red", "blue", "green"];
```

Comment:
Attaching @andrewmwells-amazon's comment to the text for easier discussion threading.
> Re-reading this, I see we use both "red" and "TinyTodo" in our examples. While I don't think the language should enforce a capitalization system, I think we should decide on, encourage and consistently use one in our examples. Based on C++ conventions, I'd vote for ALLCAPS, but I'm less opinionated on what we choose than that we should choose something.

Reply:
I also would prefer consistency here. To match "TinyTodo", maybe switch the example to use "Red", "Blue", "Green"?